### PR TITLE
Grid2.4 release note

### PR DESCRIPTION
**Pull request by sasha-astiadi**
please review and merge to staging for testnet .
will publish on production once we have 2.4 live on mainnet.

### CODE REVIEW

File: src/release_notes/release_notes_2.4.0.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# TF Grid 2.4.0 Release Note
+
+Live on testnet - Monday, January 11, 2021.
+
+## New on this release
+
+### Introducing: edge Virtual DataCenter (eVDC) v0.1.0
+  - Deploy containerized Unix IT applications on the grid via eVDC Deployer
+  - Set up Kubernetes clusters that are able to orchestrate and manage containers running on the TF Grid via eVDC admin panel.
+  - Easily extend your Kubernetes container capacity
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:10]: Extend Kubernetes capacity (as in extension of adding more worker nodes)

File: src/release_notes/release_notes_2.4.0.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# TF Grid 2.4.0 Release Note
+
+Live on testnet - Monday, January 11, 2021.
+
+## New on this release
+
+### Introducing: edge Virtual DataCenter (eVDC) v0.1.0
+  - Deploy containerized Unix IT applications on the grid via eVDC Deployer
+  - Set up Kubernetes clusters that are able to orchestrate and manage containers running on the TF Grid via eVDC admin panel.
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:9]: these containers are running on the Kubernetes cluster not on the grid

File: src/release_notes/release_notes_2.4.0.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  - Easily extend your Kubernetes container capacity
+  - add orupgrade your eVDC plan by using the built-in wallet inside your eVDC admin panel.
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:11]: can't upgrade VDC plan

File: src/release_notes/release_notes_2.4.0.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  - Deployable using TFT (ThreeFold Tokens).
+
+
+### Introducing: TF NOW Marketplace v2.4.0
+- Easy-to-deploy decentralized applications deployment via eVDC
+- The first set of 13 deployable solutions: Discourse, Gitea, Mattermost, and many more.
+- NEW: Digibyte and Dash Blockchain Nodes now deployable via eVDC TFNOW marketplace.
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:18]: to be removed (these won't be available until https://github.com/threefoldtech/vdc-solutions-charts/issues/49 they are good enough)

File: src/release_notes/release_notes_2.4.0.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- Easy operation via ‘connect’ and ‘disconnect’ button.
+- The app remains in system tray to maintain the connection. Connect or disconnect at any time.
+- Release Note: [TF Network Connector v0.1.0 beta](https://github.com/threefoldtech/yggdrasil-desktop-client)
+
+## Details of Improvements
+
+### JS-SDK 11.1
+- Separate between testnet and mainnet 3Bot identities clearly.
+- Show the monthly amount of CUs and SUs on 3Bot admin panel
+- Accept IPV4 connection to deploy 3Bot
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:52]: remove this (it's the search criteria that been relaxed not needed to be mentioned here)

File: src/release_notes/release_notes_2.4.0.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+- Provide 3Bot information on the admin panel
+- Provide TF Explorer test coverage and further test coverage on js-sdk
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:54]: remove anything related to tests, as in the last release @robvanmieghem commented it's not really relevant to release notes to mention that we tested the software more or added new tests

File: src/release_notes/release_notes_2.4.0.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+- Provide ssh access
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:55]: Provide SSH access to the deployed 3bot

File: src/release_notes/release_notes_2.4.0.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+- 3Bot monitoring system made available
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:56]: This system is part of the user VDC controller 3bot, which is enabled, it's a specific to the VDC deployments, for general 3bots for deploy3bot.grid.tf this doesn't exist

File: src/release_notes/release_notes_2.4.0.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+- Release Note: [JS-NG 11.1](https://github.com/threefoldtech/js-ng/releases)
+
+### ZOS 4.0.8
+- Provide IPv4 support for Kubernetes machines
+- Provide node metrics as a part of monitoring
+- Provide Kubernetes virtual machine monitoring.
+- Improved provisioning mechanism to avoid double provisioning of the same workload in case of explorer glitches.
+- Release Note: [ZOS 4.0.8](https://github.com/threefoldtech/zos/releases/tag/v0.4.8)
+
+### TF Explorer 0.4.8
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:75]: Important note is usage of stellar mainnet TFT on all networks in the explorer now

File: src/release_notes/release_notes_2.4.0.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+- Separate between testnet and mainnet 3Bot identities clearly.
+- Show the monthly amount of CUs and SUs on 3Bot admin panel
+- Accept IPV4 connection to deploy 3Bot
+- Provide 3Bot information on the admin panel
+- Provide TF Explorer test coverage and further test coverage on js-sdk
+- Provide ssh access
+- 3Bot monitoring system made available
+- Auto-extend capacity pools is enabled by default
+- Release Note: [JS-SDK 11.1](https://github.com/threefoldtech/js-sdk/releases)
+
```
Comment (xmonader) [inline src/release_notes/release_notes_2.4.0.md:59]: Please add two more things

- Improvements in network stability of the 3bots (detailed implementation info is embedding TRC component in the 3bot container directly)
- More hardening for stellar payments

**Review by sasha-astiadi — COMMENTED** [review]
updatre home to 2.4

**Pull request merged**